dashboard-ui/src/components/AggregateResults/aggregateResults.jsx

```diff
@@ -117,55 +241,106 @@ export default function AggregateResults() {
+                                {fullData.map((dataSet, index) => {
+                                    return (<tr key={dataSet['ParticipantID'] + '-' + index}>
+                                        {HEADER.map((val) => {
+                                            return (<td key={dataSet['ParticipantID'] + '-' + val}>
+                                                {dataSet[val] ?? '-'}
+                                            </td>);
+                                        })}
+                                    </tr>);
                                 })}
-                            </tr>);
-                        })}
-
-                    </tbody>
-                </table>
-            </div>}
-
-            {aggregateData && <table className='miniTable'>
-                <thead>
-                    <tr>
-                        <th>Mean Prop Trust</th>
-                        <th>Mean Delegation</th>
-                        <th>Mean Trust</th>
-                    </tr>
-                </thead>
-                <tbody>
-                    <tr>
-                        <td>{getMean('PropTrust')}</td>
-                        <td>{getMean('Delegation')}</td>
-                        <td>{getMean('Trust')}</td>
-                    </tr>
-                </tbody>
-            </table>}
+
+                            </tbody>
+                        </table>
+                    </div>
+                </div>
+            }
+
+            {type === 'Program' && <ProgramQuestions chartData={chartData} kdmaScatter={kdmaScatter} allData={fullData} />}
+            {(type === 'HumanProbeData' && aggregateData) && 
+                <div className="home-container">
+                    <div className="home-navigation-container">
+                        <div className="evaluation-selector-container">
+                            <div className="evaluation-selector-label"><h2>Human Simulator Probe by Environment</h2></div>
+                        </div>
+                        <div className="aggregate-button-holder">
+                            <button onClick={exportHumanSimToExcel} className='aggregateDownloadBtn'>Download Human Sim Data</button>
+                        </div>
+                    </div>
+
+                    
+                
+                    {aggregateData["groupedSim"]!== undefined && Object.keys(aggregateData["groupedSim"]).map((objectKey, key) => 
+                        <div className='chart-home-container' key={"container_" + key}>
+                            <div className='chart-header'>
+                                <div className='chart-header-label'>
+                                    <h4 key={"header_" + objectKey}>{objectKey[0].toUpperCase() + objectKey.slice(1)}</h4>
+                                </div>
+                            </div>
+                            <div key={"container_" + key} className='resultTableSection result-table-section-override'>
```

Comment:
On this line, removing 'result-table-section-override' fixed the problem @kaitlyn-sharo  mentioned for me. I think the important flags in that class in aggregateResults.css may have been causing the problem.  The only other effect I noticed from removing those styles was the rounded border on the table.

Before removing:
<img width="1372" alt="Screenshot 2024-04-09 at 4 33 43 PM" src="https://github.com/NextCenturyCorporation/itm-evaluation-dashboard/assets/73036954/8da1c5ad-de0f-4c08-b8f5-904ff5ace57f">

After Removing:
<img width="1373" alt="Screenshot 2024-04-09 at 4 34 44 PM" src="https://github.com/NextCenturyCorporation/itm-evaluation-dashboard/assets/73036954/12caa9b2-d1ef-4143-be3f-7b11def72191">

Reply:
Yeah, probably the padding in result-table-section-override. Maybe see if you can get the same effect with margin instead.

Reply:
This should all be resolved now.  Please let me know.

Reply:
Resolved! Looks great